jflex/src/main/java/jflex/Options.java

```diff
@@ -1,115 +0,0 @@
-/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
```

Comment:
Is there a reason to rename it to `GeneratorOptions`? (I don't mind much either way)

Reply:
Yes:
1. commons-cli also has Options.
2. it was easier in the migration path

In the long term, probably not a must, even if option is a bit an overloaded term.

Reply:
Ok, no worries, I'm fine with `GeneratorOptions`.